engine/org.eclipse.birt.report.engine.emitter.pdf/src/org/eclipse/birt/report/engine/emitter/pdf/README.md

```diff
@@ -33,6 +33,17 @@ The following list get an overview of all supported user properties, the content
 	Since      	4.16
 	Designer  	4.17
 
+
+**PdfEmitter.UAConformance**
+
+	Content    	define the PDF/UA conformance of the created PDF document, e.g. PDF.UA-1
+	Location   	report
+	Data type  	string
+	Values     	PDF.Standard, PDF.UA-1
+	Default    	PDF.Standard
+	Since      	4.16
```

Comment:
The version numbers of "Since" & "Designer" is 4.18 and mark the availability of the configuration option.

Reply:
> mark the availability of the configuration option.

I don't know what you mean.

Reply:
The version numbers have to be changed

- Since 4.18
- Designer 4.18

Reply:
If this doesn’t land for the December release it will change again.